### ISSUE TITLE
This css rule doesn't seem to be used

### ISSUE THREAD

**Issue by spiritinlife** [geomm/vite-vue3-pinia-typescript#15]
https://github.com/geomm/vite-vue3-pinia-typescript/blob/237d2e3b3027682f4221d88a75d9da8890f3a108/rick-and-morty-UI/src/App.vue#L26

Also in Readme tasks there is a reference for a NavComponent that we couldn't find. Are these related?

**Comment by geomm** [comment]
@spiritinlife you are correct, the style shouldn't be there. The redundant style and the reference of the NavComponent are not related.

About the nav style, I was thinking of using a global Navigation element with "Characters" link in order to use it as "back to listing" action from the details view but I removed it since the logo does exactly this job. I missed the redundant style there.

With regards to the NavComponent it is absolutely misleading - I was thinking of the Pagination Navigation in the Listing view. I missed that too. I am referring to **[ListPaginationComponent.vue](https://github.com/geomm/vite-vue3-pinia-typescript/blob/main/rick-and-morty-UI/src/components/ListingComponent/ListPaginationComponent.vue)** .
